burn-compute/src/tune/tuner.rs

```diff
@@ -15,15 +15,16 @@ use crate::client::ComputeClient;
 use crate::server::ComputeServer;
 use crate::tune::{AutotuneOperation, AutotuneOperationSet, TuneBenchmark, TuneCache};
 
-#[derive(Debug, Default)]
+#[derive(Debug)]
 /// Executes autotune benchmarking and caching
 pub struct Tuner<S: ComputeServer, C> {
     tune_cache: TuneCache<S::AutotuneKey>,
     _channel: PhantomData<C>,
 }
 
+#[allow(clippy::new_without_default)]
 impl<S: ComputeServer, C: ComputeChannel<S>> Tuner<S, C> {
-    /// Returns a tuner with empty cache
+    /// Returns a tuner with cache initialized from persistent cache
     pub fn new() -> Self {
```

Comment:
I think you will need to implement `Default` to satisfy clippy:

```rust
impl<S: ComputeServer, C: ComputeChannel<S>> Default for Tuner<S, C> {
    fn default() -> Self {
        Self::new()
    }
}
```

When providing an empty public `new` method, it's also a good idea to implement `Default`.

Reply:
I added the attribute `#[allow(clippy::new_without_default)]` 
Is that OK or is it a strong convention to implement a default as well ?